src/Npgsql/NpgsqlTypes/NpgsqlTypes.cs

```diff
@@ -268,10 +271,13 @@ public override string ToString()
 /// <summary>
 /// Represents a PostgreSQL Polygon type.
 /// </summary>
-public struct NpgsqlPolygon : IList<NpgsqlPoint>, IEquatable<NpgsqlPolygon>
+public readonly struct NpgsqlPolygon : IList<NpgsqlPoint>, IEquatable<NpgsqlPolygon>
```

Comment:
I don't think so, since it already didn't expose any APIs for mutating... But I think it's a good idea not to backport this specific change to the patch releases...

Reply:
OK, not an issue then